terraform_egress_pub/scripts/egress_pub.py

```diff
@@ -51,18 +51,10 @@
 # The following CIDR blocks are where our Qualys web application scans
 # originate from:
 qualys_was_ips = (
-    "64.39.99.19/32",
-    "64.39.99.20/30",
-    "64.39.99.24/31",
-    "64.39.99.98/31",
-    "64.39.99.100/32",
-    "64.39.99.131/32",
-    "64.39.99.132/30",
-    "64.39.99.136/29",
-    "64.39.99.144/31",
     "64.39.108.103/32",
     "64.39.108.104/29",
     "64.39.108.112/28",
+    "64.39.99.0/24",
```

Comment:
Keeping these ordered as they used to be (since the order does not impact functionality).

```suggestion
    "64.39.99.0/24",
    "64.39.108.103/32",
    "64.39.108.104/29",
    "64.39.108.112/28",
```

Reply:
I don't Know if it is a bug, but I am changing the title and notes on the commit, but it doesn't seem to hold when I confirm

Reply:
> I don't Know if it is a bug, but I am changing the title and notes on the commit, but it doesn't seem to hold when I confirm

The commit message looks good to me for https://github.com/cisagov/cyhy_amis/pull/353/commits/e75124c034c2aa78d5d4d30f0ab3a87bab548b36.  Or are you talking about a different commit?